src/main/java/DirectoryCheck.java

```diff
@@ -0,0 +1,41 @@
+package main.java;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class DirectoryCheck {
+    public static void main(String[] args) {
+        String directoryPath = "/home/buga1chuk/info.coding/test";
```

Comment:
Do not use the absolute path. It is a bad practice.
Find a solution to handle it.